.github/workflows/commit.yml

```diff
@@ -37,6 +37,11 @@ jobs:
           echo "::stop-commands::jest-output"
           npm test
           echo "::jest-output::"
+        env:
+          NEXUS_USERNAME: ${{secrets.NEXUS_USERNAME}}
+          NEXUS_PASSWORD: ${{secrets.NEXUS_PASSWORD}}
+          RUNNER_TEMP: /tmp/github_actions/temp
```

Comment:
We shouldn't need to define these ourselves. Any RUNNER_* env vars should be set automatically by the runner.

Reply:
Have a look at https://github.com/actions/toolkit/blob/947ba5b559e0924b6f153513c8001124239b6fce/packages/tool-cache/src/tool-cache.ts#L34 if you don't set the runner variables it will try to create directories in /Users on osx and /home on linux. Of course that's not going to work due to file system permissions, and it's definitely the wrong place for temporary files.